docs/reference/apilinks.rst

```diff
@@ -17,3 +17,5 @@ The following table documents all these links and their format:
      - ``/api/v2/compilereport/<compile_id>``
    * - Managed resource
      - ``/api/v2/resource/<resource_id>``
+   * - Service instance
+     - ``/lsm/v1/service_inventory/<service_entity>/<service_id>``
```

Comment:
I think this is only relevant for the iso version of the docs. I know we use the `.. only:: iso` directive in some places, but I don't know how nice it plays when used inside a `list-table` directive.

Reply:
I split this table into iso and oss. I was not able to get the `.. only:: iso` to work with the table